client/landing/gutenboarding/components/domain-picker/style.scss

```diff
@@ -32,23 +119,31 @@
 	&.components-button {
 		display: flex;
 		justify-content: space-between;
-		margin-top: 18px;
+		width: 100%;
+		height: 46px;
+		border-radius: 0;
+		padding: 0 16px;
+		font-size: 14px;
+
+		&.is-tertiary {
+			color: var( --color-text );
+
+			&:not( :disabled ):not( [aria-disabled='true'] ):hover {
+				background: var( --color-surface-backdrop );
+				color: var( --color-text );
+			}
+		}
 	}
-}
 
-.domain-picker__suggestion-item-name {
-	&.placeholder {
-		@include placeholder();
+	+ .domain-picker__suggestion-item {
+		border-top: 1px solid $light-gray-500;
```

Comment:
This shouldn't be `$light-gray-500`, but rather something from the color variables. Will rollout another commit later.